demo.md

```diff
@@ -0,0 +1,19 @@
+Currently Rodio is by far the most widely used audio crate. Unfortunately it
+seems limited by what we knew 9 years ago about audio development in Rust. Since
+then new crates have been build to overcome that. We hope to demonstrate one
+crate to be best suited to all use cases and then build support behind it. For
+that we are building a demo. It will take the form of a game environment since
+game audio is seen as the most challenging use case.
+
+Currently two crates are participating in this demo: `Rodio` and `Firewheel`
+though everyone is very welcome. The demo will be limited to functionality 
+implemented by all crates which could not be dramatically improved in the future.
+
+# Demo 
+- audio environment playing back many sounds at the same time.
+- sounds and effects are toggled on and off dynamically during the demo.
+- the demo uses audio libraries in the same way as a game.
+- pre-defined events instead of user input to keep the demo repeatable.
```

Comment:
One of the metrics is feedback latency in response to user actions, if all events are predetermined it makes it harder to judge that; humans are on average pretty poor at judging the synchronization of audio and visual elements due to how the brain compensates timing differences (i.e. correctly correlates visual and audio events even when far away and sound propagation is non-negligible).

Reply:
that makes sense. Would a single sound as response to a button press work? 

Could it work to monitor cpal for underflows?

Reply:
Numbers are good, but the point is not so much to get some measurement that proves we have to move away from Rodio, it's also to show users this is an improvement.

My opinion is, as far as the demo is concerned, the qualitative is as important as the quantitative. The demo needs to be interactive so that users can feel the change for themselves.

Reply:
> it's also to show users this is an improvement.

I'll try and work that into the wording.

Reply:
should be good now